.github/workflows/ci.yml

```
@@ -2,6 +2,7 @@ name: CI
 
 permissions:
   contents: read
+  dependency-graph: write
```

actionlint (1.7.11)

[error] unknown permission scope "dependency-graph". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)